SUMMARY.md

```diff
@@ -93,7 +93,7 @@
 
 * [Kubecost API Directory](apis-overview.md)
   * [Allocation API](allocation.md)
-  * [V2 Filters](filters-v2.md)
+  * [Allocation API Filters (v2)](filters-api.md)
```

Comment:
Reminder to change `summary.md` once you change the other doc's title!

Also since V2 filters doesn't apply solely to the Allocation API anymore ... should this doc be moved to the bottom of the Kubecost API Directory? (instead of being directly under the Allocation API doc)

Reply:
Adding linting doc titles vs summary.md to the backlog for PR checks. 😁